az-digital/az_quickstart#2330, az-digital/az_quickstart#2331 Update CONTRIBUTING.md with changed slack channel names. (#5)
az-digital/.github: .github/CONTRIBUTING.md

  * A [GitHub account](https://github.com/join).
  * [Slack](https://uarizona.slack.com) is our main source of communications.
    * Use the `#azdigital-quickstart` channel for questions/comments related to this
      project.
    * Use the `#azdigital-meetings` channel to ask questions or get updates related 

>       to Arizona Digital meetings, both physical and via Zoom.

Reviewer: ```suggestion
      to our weekly Arizona Digital meetings on Zoom.
```
